eng/pipelines/aggregate-reports.yml

```diff
@@ -30,20 +30,24 @@ jobs:
       - download: net-core
         artifact: packages
         patterns: '*'
-      - task: securedevelopmentteam.vss-secure-development-tools.build-task-credscan.CredScan@2
+      - task: securedevelopmentteam.vss-secure-development-tools.build-task-credscan.CredScan@3
         displayName: 'Run CredScan'
         inputs:
-          toolMajorVersion: V2
-          outputFormat: sarif
-          suppressionsFile: eng/CredScanSuppression.json
-      - task: ms-codeanalysis.vss-microsoft-security-code-analysis-devops.build-task-publishsecurityanalysislogs.PublishSecurityAnalysisLogs@2
-        displayName: 'Publish Security Analysis Logs'
-        inputs:
-          CredScan: true
-      - task: securedevelopmentteam.vss-secure-development-tools.build-task-postanalysis.PostAnalysis@1
+          debugMode: false
+      - task: securedevelopmentteam.vss-secure-development-tools.build-task-postanalysis.PostAnalysis@2
         displayName: 'Post Analysis'
         inputs:
-          CredScan: true
+          GdnBreakAllTools: false
+          GdnBreakGdnToolCredScan: true
+          GdnBreakGdnToolCredScanSeverity: Error
+          GdnBreakBaselineFiles: '\\eng\dotnet.gdnbaselines'
```

Comment:
Is it easy to create one of these baseline files?

Reply:
I have created the baseline files. However, I have to test how they setup the path. It seems post analysis has trouble to read the file.